topics/ecology/tutorials/QGIS_Web_Feature_Services/tutorial.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: tutorial_hands_on
+title: QGIS Web Feature Services
+questions:
+- How to use WFS webservice to access and import GIS data through Galaxy
+objectives:
+- Deal with QGIS execution management
+- Access, filter, and import GIS data using WFS webservice
+time_estimation: 0H30M
+key_points:
+- From WFS webservice, you can now access GIS remote data, build a query to select a subpart of the data, and import it to Galaxy through QGIS Galaxy interactive tool.
+tags:
+  - GIS
+  - Geographical Information System
+  - WFS
+  - Spatial data
+  - QGIS
+  - Maps
+  - OGC
+contributors:
+- colineroyaux
+- Marie59
+- yvanlebras
+
+---
+
+
+# Introduction
+
+Based on this [QGIS official tutorial](https://docs.qgis.org/2.18/en/docs/training_manual/online_resources/wfs.html), you will learn here how to access, filter and import GIS data through WFS web service using QGIS Galaxy interactive tool:
+
+In the Geographical Information System landscape, there is existing standards to help users deal with remote data. The most common web services are Web Map Services (WMS) and Web Feature Services (WFS). If WMS allows users only to access and display maps stored remotely, WFS is giving access to the features of data so you can modify it and create your own data and maps.
```

Comment:
https://training.galaxyproject.org/training-material/news/2021/05/25/abbreviations-tag.html might be useful here